.pre-commit-config.yaml

```diff
@@ -73,6 +73,12 @@ repos:
     hooks:
       - id: black
         language_version: python3
+  - repo: https://github.com/mozilla-l10n/moz-fluent-linter
+    rev: v0.4.4
+    hooks:
+      - id: fluent_linter
+        files: \.ftl$
+        args: [--config, .github/l10n/linter_config.yml, l10n/en/]
```

Comment:
we also have FTL files in `l10n/en-US` and `l10n-pocket/en`. Could we add those?